### DISCUSSION TITLE
希望可以隐藏 动态的数字角标提醒

### DISCUSSION THREAD

**Discussion by AugustTuan** [the1812/Bilibili-Evolved#4125] [功能建议 / Ideas]
如图，每次登录或者隔一段时间刷新页面，总会弹出这个数字角标，吸引我去查看新的动态
希望可以隐藏掉这个数字角标的提醒
![Snipaste_2023-04-02_20-51-14](https://user-images.githubusercontent.com/37429029/229411497-e13c9b3f-1cf0-470c-a11b-c4ff07fa612f.png)

**Comment by Yuminis98** [comment]
支持，我也不想每次都看到这个动态的角标

**Comment by Rick0525** [comment]
支持！能做不？

**Reply by Geeyun-JY3** [reply]
有没有一种可能，给这个 discussion 打的这个 `feat: completed` 的 label 就是为了告诉你这功能已经做了：#1297

> 如果是 2021-10-14 之后提出的功能请求类, 我会尝试使用 Tag 标明状态:
> 
> - 没 Tag: 还在调研中
> - `feat: reviewed`: 已了解此建议 (但不一定是马上做, 我空闲时间有限, 只挑一些感兴趣的做)
> - `feat: completed`: 已实现此建议
> - `feat: rejected`: 不会实现此建议

![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/edab12a3-2f33-4cad-850e-571a05b31982)
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/a3ea33db-f93c-4c5d-8da3-5709676a3a84)

**Reply by Rick0525** [reply]
谢谢你！对不起上午没有认真看标签，也没有找到这个功能。然后我记得在别的Repository的issue里，更改label会在消息列表中有一行的，为啥这里没有呢？

**Reply by the1812** [reply]
这个不是 issue, tag 变更没啥提醒

**Reply by Rick0525** [reply]
谢谢你！
